driver/src/main/java/eu/cloudnetservice/cloudnet/driver/event/EventManager.java

```diff
@@ -62,9 +61,9 @@
   @NonNull EventManager unregisterListener(Object @NonNull ... listeners);
 
   /**
-   * Calls the given event to the {@code *} channel, triggering all event listeners which are listening to it.
+   * Calls the given event to the * channel, triggering all event listeners which are listening to it.
    * <p>
-   * This method call is equivalent to {@code callEvent("*", event)}.
+   * This method call is equivalent to callEvent("*", event).
```

Comment:
`@code` was correct here, maybe change this to `{@code manager.callEvent("*", event)}`